.github/workflows/main.yaml

```
@@ -0,0 +1,32 @@
+name: Automated testing
+
+on:
+  push:
+    branches:
+      - "*"
+      - "!test*"
```

`*` can't match `/`, which might appear in a branch name from forks. Perhaps you want `**` to match `/` too here. See https://github.com/latex3/latex3/commit/62c5afee35e44d3a9db4c9432103349251a92a1c and related [GitHub Actions doc](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet).

And, need `pull_request` event here?

```suggestion
on:
  push:
    branches:
      - "**"
      - "!test**"
  pull_request:
```